students/kristianjf/lesson02/series.py

```diff
@@ -0,0 +1,64 @@
+def Fibonacci(n):
+    '''
+    Step 1
+    Create a new module series.py in the lesson02 folder in your student folder.
+    In it, add a function called fibonacci.
+    The function should have one parameter, n.
+    The function should return the nth value in the fibonacci series.
+    Ensure that your function has a well-formed docstring
+    '''
+    n = abs(n)
+    n1 = 0
+    n2 = 1
+    fib_list = [n1, n2]
+    for x in range(1,n):
+        n3 = n1 + n2
+        fib_list.append(n3)
+        n1 = fib_list[-2]
+        n2 = fib_list[-1]
+    return fib_list, fib_list[n]
+
+print('Fibonacci List: {} \nFibonacci Value: {}'.format(Fibonacci(7)[0],Fibonacci(7)[1]))
+
+def Lucas(n):
+    '''
+    The Lucas Numbers (Links to an external site.)Links to an external site. are a related series of integers that start with the values 2 and 1
+    '''
+    n = abs(n)
+    n1 = 2
+    n2 = 1
+    luc_list = [n1, n2]
+    for x in range(1,n):
+        n3 = n1 + n2
+        luc_list.append(n3)
+        n1 = luc_list[-2]
+        n2 = luc_list[-1]
+    return luc_list, luc_list[n]
+
+print('Lucas List: {} \nLucas Value: {}'.format(Lucas(7)[0],Lucas(7)[1]))
+
+def sum_series(n, n1=0, n2=1):
```

Comment:
Good, I like the logic. It's clean and easy to follow. 
A few comments: 
For all these, we really only needed the nth value of the series, so storing all the values in a sum_list can get computationally costly if we ask for a really high number. Another option would be to store the total sum and only hold onto n1 and n2 as variables. Something kind of like this:
```
def fib2():
    a = 0
    b = 1
    so_far = 0
    for i in range(10):
        so_far += a
        a, b = b, a + b
    print(so_far)
fib2()
```